README.md

```diff
@@ -1,2 +1,27 @@
 # joining-18f-app
 The 18F application form
+
+## Installing
+
+After you've cloned this repo and made a Virtualenv with Python 3.x, run:
+    
+    pip install -r requirements.txt
```

Comment:
On my machine it has to be `pip3 install -r requirements.txt`. Varies by local environment config, I'm sure, but I'd go for that one.